### PR TITLE
improve NAS quickstart

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: docs/en_US/NAS/QuickStart.rst [hunk 1]
```diff
@@ -24,97 +24,90 @@ Define Base Model
...
+    def forward(self, x):
+      x = F.relu(self.conv1(x))
+      x = F.max_pool2d(self.conv2(x), 2)
+      x = torch.flatten(self.dropout1(x), 1)
+      x = self.fc2(self.dropout2(F.relu(self.fc1(x))))
+      output = F.log_softmax(x, dim=1)
+      return output
 
-  .. code-block:: python
+This example uses two mutation APIs, ``nn.LayerChoice`` and ``nn.ValueChoice``. ``nn.LayerChoice`` takes a list of candidate modules (two in this example), one will be chosen for each sampled model. It can be used like normal PyTorch module. ``nn.ValueChoice`` takes a list of candidate values, one will be chosen to take effect for each sampled model.
```
Comment (ultmaster) [inline docs/en_US/NAS/QuickStart.rst:96]: Missing blank line here.

File: docs/en_US/NAS/QuickStart.rst [hunk 2]
```diff
@@ -24,97 +24,90 @@ Define Base Model
...
       super().__init__()
-      self.conv = nn.Conv2d(32, 1, 5)  # possibly mutate this conv
-      self.pool = nn.MaxPool2d(kernel_size=2)
-    def forward(self, x):
-      return self.pool(self.conv(x))
+      self.conv1 = nn.Conv2d(1, 32, 3, 1)
+      self.conv2 = nn.Conv2d(32, 64, 3, 1)
+      self.dropout1 = nn.Dropout(0.25)
+      self.dropout2 = nn.Dropout(0.5)
+      self.fc1 = nn.Linear(9216, 128)
```
Comment (ultmaster) [inline docs/en_US/NAS/QuickStart.rst:44]: 9216 looks like a very large number. You might want to change the conv stride to 2.

**Review by scarlett2018 — APPROVED** [review]

File: docs/en_US/NAS/QuickStart.rst (hunk 1 above)
Comment (QuanluZhang) [reply docs/en_US/NAS/QuickStart.rst:96]: why? there is blank line before and after this paragraph

File: docs/en_US/NAS/QuickStart.rst (hunk 2 above)
Comment (QuanluZhang) [reply docs/en_US/NAS/QuickStart.rst:44]: let's keep it for now. I will update it when working on retiarii experiment tasks.

File: docs/en_US/NAS/QuickStart.rst (hunk 1 above)
Comment (ultmaster) [reply docs/en_US/NAS/QuickStart.rst:96]: Oops. Maybe there is something wrong with my eyes.

**Review by ultmaster — APPROVED** [review]

**Pull request merged**